### DISCUSSION TITLE
BUG: some GPL licenses show on UI as LGPL

### DISCUSSION THREAD

**Discussion by hughperkins** [community/community#16241] [Other Feature Feedback, Questions, & Ideas]
This license is clearly GPL, yet shows up as LGPL in the github interface:

https://github.com/zachjs/iverilog/blob/master/COPYING

<img width="1365" alt="Screen Shot 2022-05-09 at 5 27 02 AM" src="https://user-images.githubusercontent.com/123560/167381530-0a4c88f7-d791-4a60-9a2f-9001a425224f.png">

This seems pretty critical as a bug, since LGPL gives one radically different rights than GPL.

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
